src/tests/compiletest.md

```diff
@@ -11,6 +11,14 @@ efficient test execution (parallel execution is supported),
 and allows the test author to configure behavior and expected results of both
 individual and groups of tests.
 
+> NOTE:
+> For macOS users, `SIP` (System Integrity Protection) [may consistently
+> check the compiled binary by sending network requests to Apple](https://rust-lang.zulipchat.com/#narrow/stream/182449-t-compiler.2Fhelp/topic/.E2.9C.94.20Is.20there.20any.20performance.20issue.20for.20MacOS.3F), so you may
```

Comment:
Could you use the reference-style syntax (`[word]: https://xxx.com`)? The line is long and it may be troublesome when editing in the future.

Reply:
fixed